tests/test_octodns_record.py

```diff
@@ -1275,10 +1275,18 @@ def test_base(self):
                 'value': '1.2.3.4',
             })
         reason = ctx.exception.reasons[0]
-        self.assertTrue(reason.startswith('invalid name, "xxxx'))
+        self.assertTrue(reason.startswith('invalid label, "xxxx'))
         self.assertTrue(reason.endswith('xxx" is too long at 64'
                                         ' chars, max is 63'))
 
+        # should not raise with dots
+        name = 'xxxxxxxx.' * 10
+        Record.new(self.zone, name, {
+            'ttl': 300,
+            'type': 'A',
+            'value': '1.2.3.4',
+        })
+
```

Comment:
Please add in a case where the there's some valid pieces and some that are too long, e.g.

```
`foo.` + ('x' * 65) + .bar`
```

so that we're explicitly testing for a "bad" case with dots and not just relying on the non-dot too long case up above to trigger the validation, in case the implementation changes someday.

Reply:
Good idea, done.